Cargo.toml

```diff
@@ -90,91 +90,91 @@ tiny-bip39 = { version = "1.0" }
 tokio = { version = "1.32" }
 rand_pcg = { version = "0.3.1", default-features = false }
 
-frame-benchmarking = { git = "https://github.com/Cardinal-Cryptography/substrate.git", branch = "aleph-v1.0.0", default-features = false }
```

Comment:
New repo <3 

Can you please create a ticket to make `https://github.com/Cardinal-Cryptography/substrate.git` read only. when 1.1.0 is released to production? Till then we need both repos open in case of backporting a hotfix to 1.0.0

Reply:
sure, here you go https://cardinal-cryptography.atlassian.net/browse/A0-3468

Reply:
Thanks